python/cudf/.coveragerc

```
@@ -1,4 +1,4 @@
 # Configuration file for Python coverage tests
 [run]
-include = cudf/*
-omit = cudf/tests/*
\ No newline at end of file
+source = cudf/*
```

Alternatively, you can just specify the package name:
```suggestion
source = cudf
```
> You can specify source to measure with the `--source` command-line switch, or the `[run] source` configuration value. The value is a comma- or newline-separated list of directories or importable names (packages or modules).

https://coverage.readthedocs.io/en/6.3.2/source.html#execution